src/domUpdates.js

```diff
@@ -87,40 +88,62 @@ const domUpdates = {
     ]);
   },
 
-  renderUserDashboard(user, bookings) {
+  renderUserDashboard(user, bookings, expenses) {
     let welcomeMsg = document.getElementById('welcomeMsg');
     let upcomingStays = document.getElementById('upcomingStays');
     let pastStays = document.getElementById('pastStays');
     let totalSpent = document.getElementById('totalSpent');
 
     welcomeMsg.innerText = `Welcome ${user.name}`;
-    
-    bookings.forEach(booking => {
-      let bookingDate = dayjs(booking.date)
-      console.log(bookingDate.isSame(currentDate))
 
+    bookings.forEach(booking => {
+      let bookingDate = dayjs(booking.booking.date);
       if (bookingDate.isBefore(currentDate)) {
+        let formattedDate = bookingDate.format('MMM D YYYY')
         pastStays.innerHTML += `
-        <div class="past-booking-card">
-          <h4>Date: ${booking.date}</h4>
-          <ul>Room: ${booking.roomNumber}</ul>
-        </div>
-          `;
-      } else if (bookingDate.isAfter(currentDate) || bookingDate.isSame(currentDate)) {
+          <div class="past-booking-card">
+            <div class="image-area">
```

Comment:
This looks good! Is there a different element tag from div that you could use here to make it more accessible?

Reply:
ty I'll switch to article